week-1/implement/sum.js

```diff
@@ -0,0 +1,17 @@
+function sumNumber(array) {
+    if(array.length===0){
+        return 0;
+    }
+    if(array.length===1){
+        return Number(array);
+    }
+    let sum=0;
+    for(const item of array){
+        if (typeof(item)==='number'){
+            sum+=item;
+        }
+    }return sum;
```

Comment:
As a matter of style, please ensure you put your return statements on a line on their own, just to make the code layout nice and easier to read.